config/initializers/session_store.rb

```diff
@@ -1 +1,15 @@
-Rails.application.config.session_store :active_record_store, key: '_upaya_session'
+require 'idle_session_expiration_store'
+
+options = {
+  key: '_upaya_session',
+  redis: {
+    driver: :hiredis,
+    expire_after: Figaro.env.session_timeout_in!.to_i.minutes,
+    key_prefix: "#{Figaro.env.domain_name!}:session:",
+    url: Figaro.env.redis_url!
+  },
+  # on_redis_down: ->(e, env, sid) { do_something_will_ya!(e) },
+  # on_session_load_error: ->(e, sid) { do_something_will_ya!(e) },
+  serializer: :marshal
```

Comment:
* Marshal is what the ActiveRecord session store is using [now](https://github.com/rails/activerecord-session_store/blob/master/lib/active_record/session_store.rb#L44)
* Marshal might be [slightly faster than JSON](https://gist.github.com/shilov/1691428)
* We don't store our sessions in cookies, so `config/initializers/cookies_serializer.rb` may be ignored?